docs/pages/Cardinal/API-Reference/Transactions.mdx

```diff
@@ -0,0 +1,208 @@
+# Transactions
+
+This API reference will help you understand how to use transactions and configuration options available for transactions.
+
+```go
+package transactions
```

Comment:
i think we called this package `tx` in the starter-game-template. let's try to make it consistent to reduce confusion

Reply:
[docs: rename example package from transactions to tx](https://github.com/Argus-Labs/world-engine/pull/200/commits/f495c8ad1f9bb7a9d162ce892d556a442b5bd176)